src/BaseInvoker.sol

```diff
@@ -5,37 +5,32 @@ import { Auth } from "src/Auth.sol";
 
 /// @title BaseInvoker
 /// @author Anna Carroll <https://github.com/anna-carroll/3074>
+/// @author Jake Moxey <https://github.com/jxom>
 /// @notice Shared functionality for Invoker contracts to efficiently AUTH a signer, then execute arbitrary application logic.
 /// @dev To implement an Invoker contract, simply inherit BaseInvoker and override the exec function with your Invoker logic.
 abstract contract BaseInvoker is Auth {
-    /// @notice thrown when `execute` finishes with leftover value in the Invoker contract, which is unsafe as it could be spent by anyone
-    error ExtraValue();
```

Comment:
was this removed because this sort of this sort of side effect cannot happen anymore due to changes in how eth is spent by the invoker/authorized?

Reply:
exactly, yes. ETH is spent from the authority's wallet now, not the Invoker. you don't need to send ETH into the Invoker anymore.

Reply:
related: https://github.com/anton-rs/3074-invokers/pull/17/files#r1581455979

Reply:
addressed https://github.com/anton-rs/3074-invokers/pull/19